examples/async/src/schema/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Schema
+ *
+ * this is a direct model of the previous redux data shape
+ *
+ * type Query {
+ *  postsBySubreddit(subreddit: String!): Subreddit
+ * }
+ *
+ * type Subreddit {
+ *  items: [Post]
+ *  lastUpdated: Number
+ * }
+ *
+ * type Post {
+ *  title: String
+ * }
+ *
+ *
+ */
+
+export default {
+  Query: {
+    // this is only called on a cache miss
+    postsBySubreddit: async (_, { subreddit }) => {
+      try {
+        // trigger request posts
+        const items = await fetch(`https://www.reddit.com/r/${subreddit}.json`)
+          .then(response => response.json())
+          .then(json => json.data.children.map(child => child.data))
+          .then(posts =>
+            posts.map(({ title }) => ({ title, __typename: 'Post' })),
```

Comment:
What's our plan for typechecking? Do these types correspond to types on the server schema?

Reply:
none right now haha. Which is on par with existing state solutions but I would like to have it

Reply:
let's figure out a plan in our 1:1, i think people are going to ask about it when we release